tide/core/network/MPICommunicator.cpp

```diff
@@ -44,6 +44,11 @@
 
 #include "utils/log.h"
 
+// WAR some deadlocks receiving MPI_IBcast with OpenMPI (version 1.10.2)
```

Comment:
What does WAR mean in this context?

Reply:
Haha yeah it stands for WorkARound, it's an abbreviation that was used in multiple places by Daniel and Stefan so I adopted it, despite I also didn't get it at first

Reply:
If you want then change it to workaround.